firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -52,6 +52,15 @@ final class MicrosurveyTests: BaseTestCase {
 
         XCTAssertTrue(app.images[AccessibilityIdentifiers.Microsurvey.Survey.firefoxLogo].exists)
         XCTAssertTrue(app.buttons[AccessibilityIdentifiers.Microsurvey.Survey.closeButton].exists)
+        let tablesQuery = app.scrollViews.otherElements.tables
+        let firstOption = tablesQuery.cells.firstMatch
+        firstOption.tap()
+        XCTAssertEqual(firstOption.label, "Very satisfied")
+        XCTAssertEqual(firstOption.value as? String, "1 out of 6")
+
+        let secondOption = tablesQuery.cells["Neutral"]
+        XCTAssertEqual(secondOption.label, "Neutral")
```

Comment:
```suggestion
        mozWaitForElementToExist(secondOption)
        XCTAssertEqual(secondOption.label, "Neutral)
        mozWaitForValueToContains(secondOption, "Unselected, 3 out of 6")
```

Reply:
It seems that " is missing from Neutral, updating.